src/gstnlaunch.c

```diff
@@ -583,10 +584,21 @@ main (int argc, char **argv)
 done:
   if (thiz->loop)
     g_main_loop_unref (thiz->loop);
+
+  for (l = thiz->branches; l; l = g_list_next (l)) {
+    g_autoptr(GstBus) bus = NULL;
+
+    branch = l->data;
```

Comment:
no need to cast here ?

Reply:
No, `gpointer` are casted automatically.